documentation/book/ref-kafka-user.adoc

```diff
@@ -139,16 +139,17 @@ Use `\*` to allow or deny the operation from all hosts.
 The `host` field is optional and when not specified, the value `*` will be used as default.
 
 `resource`:: Specifies the resource for which does the rule apply.
-Simple Authorization supports 3 different resource types:
+Simple Authorization supports 4 different resource types:
```

Comment:
4 > four

(As per IBM Style Guide: "In general, write in words numbers that are less than 10".)